Lint the entire codebase

rydente opened the pull request:
Use eslint to lint the entire codebase currently so we can have a
consistent style guide to go with from here on out.

Signed-off-by: Ryan Y <ryayak1460@protingumas.com>

Commit:
Lint the entire codebase

Use eslint to lint the entire codebase currently so we can have a
consistent style guide to go with from here on out.

Signed-off-by: Ryan Y <ryayak1460@protingumas.com>

wbprice commented on api/models/SurveyResultAnswer.js:
```diff
@@ -8,28 +8,28 @@ const Model = require('trails-model');
+                    }
+                }
+            }
+        };
```
This looks a little funky.

rydente replied:
...now that I think about it, what does this code do? Why do we make two calls using a comma instead of a semicolon?

wbprice replied:
It says that a `SurveyResult` `hasMany`  `SurveyResultAnswers` and that
a 'Question' 'hasOne' 'SurveyResultAnswer'.  Unsure why a comma is being used here.  Replace with a semicolon for sure.

rydente replied:
Done in 30c035e!

wbprice commented on api/models/SurveyResultCategory.js:
```diff
@@ -1,42 +1,42 @@
+                    }
+                }
+            }
+        };
```
Also funky.

wbprice reviewed (changes requested):
This seems approximately fine.  There are a couple of instances where `eslint --fix` appears to have mangled the formatting.  Please fix those manually.

Commit:
Replace continuation with separator

Replace the commas in two different consecutive function call sequences
with semicolons so we can physically distinguish different function
calls when configuring our model relationships.

Signed-off-by: Ryan Y <ryayak1460@protingumas.com>

wbprice commented:
@rydente does `npm test` pass now?

rydente commented:
@wbprice Yessir!
